agentarchives/archivesspace/client.py

```diff
@@ -288,15 +288,21 @@ def _format_dates(self, start, end=None):
             return start
 
     def _fetch_dates_from_record(self, record):
+        dates = self._fetch_date_expression_from_record(record)
+        if not dates:
+            try:
+                start_date = record['dates'][0]['begin']
+            except IndexError:
+                return ''
+            end_date = record['dates'][0].get('end')
+            return self._format_dates(start_date, end_date)
+
+    def _fetch_date_expression_from_record(self, record):
```

Comment:
`_fetch_date_expression_from_record` returns `''` if no 'dates' field, but `None` if no 'expression' in 'dates'. Is this intentional?